user/troubleshooting/resume-suspend-troubleshooting.md

```diff
@@ -9,6 +9,10 @@ ref: 94
 title: Suspend/Resume Troubleshooting
 ---
 
+# Troubleshooting problems relating to suspend/resume
```

Comment:
There is no need to repeat the title (already included in the `title:` above).

Reply:
Removed and rebased on master.